packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -24,6 +24,37 @@ MeetingsUtil.extractDestination = (destination, type) => {
   return dest;
 };
 
+/**
+ * Determine if the provided destination is a personId.
+ *
+ * @param {string} destination - The destination to validate.
+ * @returns {boolean} - True if the destination is a personId.
+ */
+MeetingsUtil.determineIfDestinationIsPerson = (destination) => {
+  const decodedId = Buffer.from(destination, 'base64')
+    .toString('ascii');
+
+  const destUUID = decodedId.split('/').pop();
+
+  return !!(
+    decodedId.includes('PEOPLE') &&
+    destUUID.length === 36 &&
+    destUUID.split('-').length === 5
+  );
+};
+
+/**
+ * Extract the UUID from a provided PersonId destination.
+ *
+ * @param {string} destination - PersonId to extract the UUID from.
+ * @returns {string} - The extracted UUID.
+ */
+MeetingsUtil.extractUUIDFromHydraDestination = (destination) => Buffer
```

Comment:
Using `deconstructHydraId`, you won't need this function at all 😄

Reply:
made this change.